inst/pages/Introductory-workflow/Introductory.qmd

```diff
@@ -0,0 +1,293 @@
+                              name = c("shannon"))
+```
+
+We can also compare the two measures of diversity by producing the following graphs.
+
+```{r}
+library(patchwork)
+
+# Create the plots
+plots <- lapply(c("observed", "shannon"),
+                plotColData,
+                object = tse_alpha,
+                x = "patient_status",
+                colour_by = "patient_status")
+
+# Fine-tune visual appearance
+plots <- lapply(plots, "+", 
+                theme(axis.text.x = element_blank(),
+                      axis.title.x = element_blank(),
+                      axis.ticks.x = element_blank()))
+
+# Plot the figures
+(plots[[1]] | plots[[2]]) +
+  plot_layout(guides = "collect")
+
+```
+
+It's very important to make all these comparisons in order to quantify diversity and compare samples in our data using different measures. - You can find other types of comparisons directly in the book @sec-community-diversity.
+
+<hr>
+
+## Community similarity
+
+Community similarity refers to the way microorganisms resemble each other in terms of their composition and abundance of different microbial taxa. This can help us understand to what degree different samples resemble each other and finding key information. In microbiome analysis however, it's more common to measure the dissimilarity/Beta diversity between two samples A and B using the bray-Curtis measure which is defined as follows:
+
+$$
+BC_{ij} = \frac{\sum_{k} |A_{k} - B_{k}|}{\sum_{k} (A_{k} + B_{k})}
+$$ Luckily for us, the [**mia package**](https://microbiome.github.io/mia/) provides an easy way to calculate the Bray-Curtis dissimilarity and add it to our TreeSE using the `transformAssay` method:
+
+```{r}
+tse <- transformAssay(tse,
+                      assay.type = "counts",
+                      method = "relabundance")
+```
+
+This will take the original counts assay and apply the Bray-Curtis formula to it. The result is a matrix with the sample identifiers as rows and the dissimilarity to other samples as columns. It can be accessed in the assays of the tse:
+
+```{r}
+assay(tse, "relabundance")[5:10,1:10]
+```
+
+In our case, the assay contains 151 rows and 27 columns. Having so many columns and thus dimensions can be troublesome for visualizing dissimilarity.
+
+In order to visualize the dissimilarity between the different samples, we can conduct a Principal Coordinate analysis on the newly created assay. This essentially projects the Bray-curtis dimensions onto a lower space whilst maintaining as much of the variation as possible, the projected values are called principal coordinates. You can read more [**here**](https://en.wikipedia.org/wiki/Multidimensional_scaling#Types).
+
+MIA doesn't provide a direct way to reduce dimensionality but we can use Bioconductor's scater package and the vegan package, the latter can be found [**here**](https://cran.r-project.org/web/packages/vegan/vegan.pdf) to turn the dissimilarity into actual distances that can be visualized:
```

Comment:
mia, not MIA. mia has actually some dimension reduction techniques available such as dbRDA